hw/ip_templates/rstmgr/doc/theory_of_operation.md.tpl

```diff
@@ -134,7 +134,7 @@ The reset manager then checks as follows:
 
 - If all reset conditions are satisfied, wait for the reset release to gracefully complete the cycle.
 
-% if with_alert_handler:
+% if with_alert_dump:
```

Comment:
We discussed the reset indications are independent of handling an alert dump, so line 185 in rstmgr.hjson.tpl is not conditioned by `with_alert_dump`. On the same vein, this section should not be conditioned.